src/SourceGenerators/Uno.UI.SourceGenerators.Tests/DependencyObjectGeneratorTests/Given_DependencyObjectGenerator.cs

```diff
@@ -169,20 +169,14 @@ private void __InitializeBinder()
 	 		/// </summary>
 	 		[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
 	 		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
-	 		public void ClearBindings()
-	 		{
-	 			__Store.ClearBindings();
-	 		}
+	 	
 	 		
 	 		/// <summary>
 	 		/// Obsolete method kept for binary compatibility
 	 		/// </summary>
 	 		[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
 	 		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
```

Comment:
the relevant test cases would need to be updated to match the changes you made in the generator